src/Features/VisualBasic/Portable/Diagnostics/Analyzers/VisualBasicSimplifyTypeNamesDiagnosticAnalyzer.vb

```diff
@@ -82,6 +82,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.SimplifyTypeNames
                 diagnosticId = IDEDiagnosticIds.PreferIntrinsicPredefinedTypeInMemberAccessDiagnosticId
             ElseIf expression.Kind = SyntaxKind.SimpleMemberAccessExpression Then
                 Dim memberAccess = DirectCast(expression, MemberAccessExpressionSyntax)
+                Dim method = model.GetMemberGroup(expression)
+                If method.Length = 1 Then
```

Comment:
My understanding is that this returns the enclosing scope of the function which there should be a single one. Correct me if I'm wrong. 
Might need to do some more testing to verify this is actually true.